src/qt/paymentrequestplus.h

```diff
@@ -13,6 +13,17 @@
 #include <QList>
 #include <QString>
 
+// Used in getMerchant() to return state
+enum GetMerchantStatus {
+    PR_NOTINITIALIZED = 0,
```

Comment:
The `GetMerchantStatus` is a bit mask where you can combine things.
It's only a nit. But IMO combining a returned status in a bit mask is a bit wired.